tool/tsh/common/db_test.go

```diff
@@ -60,6 +60,7 @@ func TestTshDB(t *testing.T) {
 	t.Run("Login", testDatabaseLogin)
 	t.Run("List", testListDatabase)
 	t.Run("FilterActiveDatabases", testFilterActiveDatabases)
+	t.Run("GetDatabase", testGetDatabase)
```

Comment:
curious if these tests can share a setup instead of creating a new suite each test

Reply:
I tried doing that when I joined them originally, but it became a headache when changing one test's setup affects others. Test cases depend a lot on what the backend databases are, so it was just easier to keep them separate and parallel.

Since they run in parallel though, the overall test suite runs on my machine in pretty much the same time as the longest test duration